hack/build.sh

```diff
@@ -72,6 +74,7 @@ yq eval -i '
 	.metadata.name = strenv(OPERATOR_NAME) |
 	.metadata.annotations.version = strenv(TAG) |
 	.metadata.annotations.containerImage = strenv(OPERATOR_DIGEST) |
+	.metadata.labels += {"operatorframework.io/arch.amd64": "supported", "operatorframework.io/arch.s390x": "supported", "operatorframework.io/arch.ppc64le": "supported", "operatorframework.io/os.linux": "supported"} |
```

Comment:
Can we just edit and commit changes to bundle/manifests/container-security-operator.clusterserviceversion.yaml instead of doing it in the build time?

Reply:
Sure. Will do that